cpp/src/bindings.cpp

```diff
@@ -618,4 +623,13 @@ PYBIND11_MODULE(_pypowsybl, m) {
     m.def("add_network_element_properties", &pypowsybl::addNetworkElementProperties, "add properties on network elements", py::arg("network"), py::arg("dataframe"));
     m.def("remove_network_element_properties", &pypowsybl::removeNetworkElementProperties, "remove properties on network elements", py::arg("network"), py::arg("ids"), py::arg("properties"));
     m.def("get_provider_parameters_names", &pypowsybl::getProviderParametersNames, "get provider parameters for a loadflow provider", py::arg("provider"));
+
+    m.def("update_extensions", pypowsybl::updateNetworkElementsExtensionsWithSeries, "Update extensions of network elements for a given element type with a series",
+          py::call_guard<py::gil_scoped_release>(), py::arg("network"), py::arg("name"), py::arg("dataframe"));
+    m.def("remove_extensions", &pypowsybl::removeExtensions, "Remove extensions from network elements", py::arg("network"), py::arg("name"), py::arg("ids"));
```

Comment:
not very important but we could add the GIL release to the get / remove / create methods, since they could take a non-negligible execution time

Reply:
Done. Updated also the extensions' docs.